pkg/config/env.go

```diff
@@ -209,4 +209,21 @@ var Config = struct {
 	// UI path  => localhost:18000/foo"
 	// API path => localhost:18000/foo/api/v1"
 	WebPrefix string `env:"FLAGR_WEB_PREFIX" envDefault:""`
+
+	// NotifyTimeout - for each notifier that uses http.Client to make requests
+	NotifyTimeout time.Duration `env:"FLAGR_NOTIFY_TIMEOUT" envDefault:"30"`
+	// NotifyNumAttempts - how many attempts should be made for each notification in case of failure
+	NotifyNumAttempts int `env:"FLAGR_NOTIFY_NUM_ATTEMPTS" envDefault:"10"`
+	// NotifyRetryMin - the minimum amount of time that should pass between retry attempts
+	NotifyRetryMin int `env:"FLAGR_NOTIFY_RETRY_MIN" envDefault:"0"`
+	// NotifyRetryMax - the maximum amount of time that should pass between retry attempts
+	NotifyRetryMax int `env:"FLAGR_NOTIFY_RETRY_MAX" envDefault:"0"`
+
+	// Notify Webhooks
+	NotifyWebhookEnabled bool   `env:"FLAGR_NOTIFY_WEBHOOK_ENABLED" envDefault:"false"`
+	NotifyWebhookURL     string `env:"FLAGR_NOTIFY_WEBHOOK_URL" envDefault:""`
+	// Notify Slack webhooks
```

Comment:
For extendibility, I wonder if we need to support multiple channels at this time. The reason is that once we rolled out this change, it's hard to extend the naming of the env to support multiple slack channels. Maybe we should support multiple channels

Reply:
After reading some threads and the current [documentation](https://api.slack.com/messaging/webhooks) it seems to me that the channel now is kind of immutable, so one webhook is bind to a specific channel. So I think we can simplify that removing the channel en var and making the `NotifySlackURL` a slice. Which make me thing we can support multiple webhook urls and not just multiple slack urls, what do you think?